### PR TITLE
Set Input Type for TensorRT convert tests

### PR DESCRIPTION
**Pull request by eee4017**
### PR types
Others

### PR changes
Others

### Description
We preprocess the test_trt_convert test to standardize the input type to either FP32 or FP16, ensuring a uniform input type.

### CODE REVIEW

File: test/ir/inference/auto_scan_test.py
Comment (eee4017) [inline test/ir/inference/auto_scan_test.py:42]: We changed the logging system to the paddle logger.

File: test/ir/inference/auto_scan_test.py
Comment (eee4017) [inline test/ir/inference/auto_scan_test.py:749]: We reduce the input of the baseline to a smaller number of bits, which could potentially narrow the gap between the fp16 and fp32 computation.

File: test/ir/inference/program_config.py
Comment (eee4017) [inline test/ir/inference/program_config.py:70]: The name has been altered to enhance readability.

File: test/ir/inference/test_trt_convert_activation.py
Comment (eee4017) [inline test/ir/inference/test_trt_convert_activation.py:181]: For all files with the test_trt_convert test, we cast the input type of the input data in the program configuration before creating the sample configuration.

**Comment by eee4017** [comment]
Waiting for Dianhai or XiaoguangHu01 approval

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Comment by onecatcn** [comment]
@eee4017 please rerun PR-CI-APPROVAL

**Review by yuanlehome — APPROVED** [review]
LGTM

**Comment by yuanlehome** [comment]
我理解之后trt单测精度比较的容忍度更高了，之后很多单测的atol/rtol的设置是否需要批量改一下

**Comment by eee4017** [comment]
I think it could be merge

**Comment by eee4017** [comment]
I think it could be merged now.

**Pull request merged**
